tests/autorestart_test/test_container_autorestart.py

```diff
@@ -0,0 +1,101 @@
+"""
+Check the auto-restart feature of syncd, swss, teamd, bgp and dhcp_relay
+"""
+import time
+import logging
+
+from common.utilities import wait_until
+
+CONTAINER_STOP_TEST_WAITING_TIME = 30
+CONTAINER_STOP_TEST_CHECK_INTERVAL = 1
```

Comment:
Rename to `CONTAINER_CHECK_INTERVAL_SECS`, remove `CONTAINER_RESTART_TEST_CHECK_INTERVAL` and share the same constant among all tests.

Reply:
Great suggestion! Will update.